src/Integration.Vsix/SonarAnalyzerDeactivationManager.cs

```diff
@@ -10,15 +10,15 @@
 using Microsoft.VisualStudio.ComponentModelHost;
 using Microsoft.VisualStudio.LanguageServices;
 using SonarLint.Helpers;
+using SonarLint.VisualStudio.Integration.SonarAnalyzer;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
-using SonarLint.VisualStudio.Integration.SonarAnalyzer;
 
 namespace SonarLint.VisualStudio.Integration.Vsix
 {
-    public class SonarAnalyzerDeactivationManager : IDisposable
+    public class SonarAnalyzerDeactivationManager
```

Comment:
No, because it was previously doing that only because of the disposable MEF component it imported.